examples/domainmodel/example/generated/example/blog/Blog.java

```diff
@@ -0,0 +1,22 @@
+package example.blog;
+
+class Blog {
```

Comment:
I think it's better to push them. It will allow to take a look at generated files without CLI execution, directly in the repository.
So, in general we should add such things in a gitignore, but I'd prefer to push it for `Langium` usage examples.

Reply:
I see your point, but that also means we need to take care of keeping those generated files updated. I propose adding a README to the example package describing how to use the CLI.